README.md

```diff
@@ -1,6 +1,6 @@
 # Northcoders News API
 
-In order to access databases in this repo, create two env. files, test and development. Set the environment variable PGDATABASE to the appropriate database for that file. Refer to setup.sql for database names. 
+In order to access databases in this repo, create two .env.* files, test and development. Set the environment variable PGDATABASE to the appropriate database for that file. Refer to setup.sql for database names. 
 
```

Comment:
Nice work 🤸 I would possibly be a little more explicit with the naming of the .env files. For example `.env.test` and `.env.development` and maybe with the variables for each file too `PGDATABASE=database_name`